Move examples to `/examples` and adjust `Cargo.toml` accordingly (#28)
Traverse-Research/ogawa-rs: README.md

submitted for inclusion in the work by you, as defined in the Apache-2.0
license, shall be dual licensed as above, without any additional terms or
conditions.

### Examples

> Examples are located in /exampels directory a can be run by:

Reviewer: TypoS, and backticks :wink: 

```suggestion
Examples are located in the `/examples` directory and can be run by:
```

Though I'm not sure how relevant this is. This _is_ the standard crates setup for example, and would be common "cargo/Rust" knowledge :)

```
cargo run --example curves-test-vis /path/to/file.abc

cargo run --example print-tree /path/to/file.abc

cargo run --example schema-parsing /path/to/file.abc

> ```

Reviewer: I think you're missing the `--` separator here? Cargo won't understand what to do with `/path/to/...`.

Don't forget to mark the script block as `sh` or `bash`, and perhaps all 3 examples should be in a separate script block.

Author: _/path/to/file.abc_ argument works when I test it, All the examples need some .abc file path passed to them to run, so this would there to indicate that.
